platform-docs/docs/create-block/first-block-type.md

```diff
@@ -2,4 +2,55 @@
 sidebar_position: 1
 ---
 
-# Your first block type
\ No newline at end of file
+# Your first block type
+
+In addition to the built-in block types, Gutenberg offers a flexible API to build any kind of block type you can imagine.
+
+The `registerBlockType` function registers the block we are going to create and specifies the block type settings.
+
+```js
+import { createElement } from "react";
+import { registerBlockType } from '@wordpress/blocks';
+
+registerBlockType( 'create-block/gutenpride', {
+	// This is just a flag that tells the block editor that this block
+	// is using the API version 3 (the latest block type API).
+	apiVersion: 3,
+
+	// This is the display title for your block type.
+	title: 'Gutenpride',
+
+	// This is the category this block type will be listed in.
+	// Default categories include: text, media, design, widgets, theme and embed.
+	category: 'widgets',
+
+	// This is a short description for your block type.
+	// It will be shown in various places in the Gutenberg user interface.
+	description: 'Example static block scaffolded with Create Block tool.',
+
+	// This is an icon for your block type.
+	icon: (
+		<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
+			<path d="M16.7 7.1l-6.3 8.5-3.3-2.5-.9 1.2 4.5 3.4L17.9 8z" />
+		</svg>
+	),
+
+	edit() {
+		return <p>Hello from the editor!</p>;
+	},
+
+	save() {
+		return <p>Hello from the saved content!</p>;
+	},
```

Comment:
```suggestion
	edit: () => {
		return <p>Hello from the editor!</p>;
	},

	save: () => {
		return <p>Hello from the saved content!</p>;
	},
```

Probably like this?

Reply:
Well it's the same. I prefer to avoid the arrow function in this case personally.